### ISSUE TITLE
ui5-shellbar: CoPilot element has no role

### ISSUE THREAD

**Issue by elenastoyanovaa** [SAP/ui5-webcomponents#2606]
Hello @SAP/ui5-webcomponents-topic-p ,

Could you check this issue?

**Describe the bug**
ui5-shellbar-copilot-wrapper element has an aria-label and title set, but no role is applied.

**To reproduce**
1. Go to https://sap.github.io/ui5-webcomponents/master/playground/components/ShellBar/
2. Inspect the DOM of the ui5-shellbar-copilot-wrapper element

**Expected behavior**
A **role="button"** should be set to this element in order to identify the purpose of it. After implementation please update the Accessibility Status Wiki of the ui5-shellbar web component.

**Affected components**
ui5-shellbar
